roadmap.md

```diff
@@ -3,88 +3,31 @@ layout: contribute
-
-* P0. Bazel on Windows does not need to install MSYS
-* P0. Feature parity for Android, Java, C++, Python on Windows
-
-#### Migration / External repositories / Interfacing with other build systems
-
-* P0. Migration tool: Maven project to Bazel project
-* P1. Bazel can load workspace recursively
-* P1. All external repositories can use the local cache which turned on by default and has a deletion strategy
-* P1. Push to Maven public and private artifact repositories
-* P2. Repository of BUILD files for third party OSS libraries open to the community
-
-#### Skylark
-
-* P0. Skylark is fully documented; strategy for user-provided Skylark rule documentation
-* P0. Support for coverage can be extended in Skylark
-* P0. Support for testing Skylark rules
-
-#### Mobile rules
-
-* P0. Support for Android testing: integration and robolectric tests
-* P0. Bazel can build Android application on Windows
-* P0. Support for iOS testing
-
-#### Native rules
-
-* P0. Access to native rules functionality from Skylark ("sandwich")
-* P0. C++ coverage support
-
-#### Performance
-
-* P1. Local caching of build artifacts
-* P2. Bazel is benchmarked publicly by third parties.
-* P2. Minimize need for manual user config to make Bazel fast
-
-#### Misc
-
-* P1. Better CLI output (`--experimental_ui` is enabled by default)
-* P1. Full test suite is open-sourced
-* P1. Public review process for design documents
-
-#### 1.x (LTS)
-
-* P0. Google uses only public APIs of Bazel
-* P0. Github is the primary code repository
-* P1. Support policy is defined regarding LTS release
-* P1. Bazel respects the standard for Debian packaging
-* P2. Bazel is in the list of Debian package for the next stable
-
+# Bazel Roadmap
+
+## Bazel Core Improvement Areas 
+
+As of July 2018, critical areas that Bazel team addresses within Bazel core to
+fulfill the [Bazel Vision](https://docs.bazel.build/versions/master/bazel-vision.html)
+are as follows:
+
+*   **Remote execution API** is under development, but we are not yet ready to
+    provide the full guidance to rule authors on how to write rules to enable it
+    for their language.
+*   **External dependency management** story needs to be improved including both
+    the **general dependency management** mechanism in Bazel and **interfacing with
+    external package managers and build systems**. The latter is critical for
+    _gradual adoption_ of Bazel.
+*   [**Configurability**](roadmaps/configuration.html). Outside of a few
+    hardcoded cases (fat apk for Android,host -> target transitions), the promise
+    of single-command multi-platform builds is unfulfilled. 
+*   **Skylark API and tooling**. We need to support the rule authors with 
+    **clear, documented, stable API**, and **strong language and tooling**:
+    documentation generators, refactoring tools, testing frameworks and so on.
+
+## Roadmaps for specific Bazel areas
+*  [Configurability](roadmaps/configuration.html)
+*  [Windows](roadmaps/windows.html)
```

Comment:
With this change, I suggest we get rid of the menu on the left (keep only the link to the general roadmap, not the subsections).

Reply:
Add links to: skylark, external-deps, platforms